test/ng/directive/booleanAttrsSpec.js

```diff
@@ -191,34 +191,32 @@ describe('ngSrc', function() {
   }));
 
 
-  if (msie) {
-    it('should update the element property as well as the attribute', inject(
-        function($compile, $rootScope, $sce) {
-          // on IE, if "ng:src" directive declaration is used and "src" attribute doesn't exist
-          // then calling element.setAttribute('src', 'foo') doesn't do anything, so we need
-          // to set the property as well to achieve the desired effect
+  it('should update the element property as well as the attribute', inject(
```

Comment:
This is an observable change : I'm really not sure why these weren't canonicalized on IE before, but now they are, and it just seems like the right behavior.